### DISCUSSION TITLE
Lava Deep Learning 0.3.0

### DISCUSSION THREAD

**Discussion by mgkwill** [lava-nc/lava-dl#111] [Announcements]
The [_lava-dl_](https://github.com/lava-nc/lava-dl) library version 0.3.0 now enables inference for trained spiking networks seamlessly on CPU or Loihi 2 backends and can leverage Loihi 2’s convolutional network compression and graded spike features for improved memory usage and performance.

## New Features and Improvements
* Added Loihi 2 support in lava-dl NetX utilizing Loihi 2 convolution support and graded spikes (PR #88, #107).
* Added a tutorial demonstrating PilotNet application running on Intel Loihi 2 (PR #107).
* Added accelerated training of recurrent topologies in lava-dl SLAYER (PR #103)
* Added Transposed Convolution and Unpool support in lava-dl SLAYER (PR #80)

## Bug Fixes and Other Changes
* Improved lava-dl SLAYER codebase with bugfixes and additional documentation (PR #78, #105)

## Breaking Changes
- No breaking changes in this release

## Known Issues
- No known issues in this release

## What's Changed
* Remove unnecessary imports by @Tobias-Fischer in https://github.com/lava-nc/lava-dl/pull/56
* Update mnist.py by @uslumt in https://github.com/lava-nc/lava-dl/pull/71
* Updated notebooks with new hyperparameters and typo fixes by @bamsumit in https://github.com/lava-nc/lava-dl/pull/76
* Add pilotnet integration tests by @mgkwill in https://github.com/lava-nc/lava-dl/pull/79
* Slayer fixes by @bamsumit in https://github.com/lava-nc/lava-dl/pull/78
* Changes to lava-dl to reflect api changes in lava 0.4.0 by @bamsumit in https://github.com/lava-nc/lava-dl/pull/88
* Added ConvT and Unpool block for neurons by @alexggener in https://github.com/lava-nc/lava-dl/pull/80
* Update ci-build.yml, Remove redundant poetry updates by @mgkwill in https://github.com/lava-nc/lava-dl/pull/89
* Recurrent mechanic by @timcheck in https://github.com/lava-nc/lava-dl/pull/103
* Bump nbconvert from 6.5.0 to 6.5.1 by @dependabot in https://github.com/lava-nc/lava-dl/pull/93
* fix bug of `block.AbstractInput` by @fangwei123456 in https://github.com/lava-nc/lava-dl/pull/105
* Loihi Tutorials by @bamsumit in https://github.com/lava-nc/lava-dl/pull/107
* Add conda install instructions with intel-numpy by @mgkwill in https://github.com/lava-nc/lava-dl/pull/91
* Version 0.3.0 by @mgkwill in https://github.com/lava-nc/lava-dl/pull/109

## New Contributors
* @Tobias-Fischer made their first contribution in https://github.com/lava-nc/lava-dl/pull/56
* @uslumt made their first contribution in https://github.com/lava-nc/lava-dl/pull/71
* @alexggener made their first contribution in https://github.com/lava-nc/lava-dl/pull/80
* @timcheck made their first contribution in https://github.com/lava-nc/lava-dl/pull/103
* @dependabot made their first contribution in https://github.com/lava-nc/lava-dl/pull/93
* @fangwei123456 made their first contribution in https://github.com/lava-nc/lava-dl/pull/105

**Full Changelog**: https://github.com/lava-nc/lava-dl/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lava-nc/lava-dl/releases/tag/v0.3.0'>Lava Deep Learning 0.3.0</a>.</em>

**Comment by Dola47** [comment]
@mgkwill @bamsumit 

May I ask how am I supposed to cite laval-dl?

Thanks.

**Reply by bamsumit** [reply]
@Dola47 we do not have an official publication for lava-dl. You can always cite the github link. Most of lava-dl functionality is based on [SLAYER](https://papers.nips.cc/paper/7415-slayer-spike-layer-error-reassignment-in-time) paper. You can cite that if you are using `lava.lib.dl.slayer`. And we always appreciate github stars.
